### PR TITLE
Update group-mentoring.md

### PR DESCRIPTION
**Pull request by erinboyd**

### CODE REVIEW

**Comment by parispittman** [comment]
ty @erinboyd!! super small nit.

File: mentoring/group-mentoring.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Each cohort will last 2-3 months based on the time requirement in the community
 * Each mentee is working toward the same goal, cohort has an end date/timeframe  
 * Exposure to multiple areas of the project making well-rounded contributors  
 * Pilots share time commitments and responsibilities  
+* Exposure to mulitple mentors in open, collaborative environment
```
[the remark this answers is not in the document]
Comment (nikhita) [reply mentoring/group-mentoring.md]: @erinboyd can you fix that tiny nit? lgtm otherwise! :smile:

**Comment by nikhita** [comment]
I picked this in https://github.com/kubernetes/community/pull/2575. Closing.

/close

**Pull request closed without merging**